src/components/views/settings/DevicesPanel.js

```diff
@@ -218,15 +218,16 @@ export default class DevicesPanel extends React.Component {
             </AccessibleButton>;
 
         const classes = classNames(this.props.className, "mx_DevicesPanel");
+        if (SettingsStore.getValue(UIFeature.SecurityAllowSessionEdits)) return null;
```

Comment:
I am wondering why you have moved SecurityAllowSessionEdits to not show the session information in addition to not showing the delete button?

All I was suggesting in my first comment was to have a new line after the && e.g.
```
{SettingsStore.getValue(UIFeature.SecurityAllowSessionEdits) &&
<div className="mx_DevicesPanel_deviceButtons">
```

Reply:
No worries, I have reverted this. I misinterpretted your suggestion. Probably due to config cache I did not see changes locally. Now reverted.